load_testing/.env.sh.template

```
@@ -0,0 +1,3 @@
+#! /usr/bin/env sh
+
+export ACCESS_TOKEN = "<not_set>"
```

```suggestion
export ACCESS_TOKEN="<not_set>"
```

If I copy this file and add the key, the spaces result in an `./.env.sh: line 3: export: `=': not a valid identifier` error.